synapse/lib/agenda.py

```diff
@@ -366,6 +369,7 @@ def unpack(cls, stor, val):
         appt = cls(stor, val['iden'], val['recur'], val['indx'], val['query'], val['creator'], recs, nexttime=val['nexttime'], view=val.get('view'))
         appt.doc = val.get('doc', '')
         appt.name = val.get('name', '')
+        appt.pool = val.get('pool', False)
```

Comment:
we can set `pool=val.get('pool', False)` in the `cls(...)` call?

Reply:
This whole thing should be converted to an info dict, but that's a separate PR. I went with the existing convention for this one.